SignList.xml

```diff
@@ -13,6 +14,7 @@
     <ThirdParty Include="Newtonsoft.Json.dll" />
     <ThirdParty Include="Mono.Cecil.dll" />
     <ThirdParty Include="Mono.Cecil.*.dll" />
+    <ThirdParty Include="androidx*.jar" />
```

Comment:
What was the actual list of jar files? I don't think this should be in that list.

Reply:
I see what these are. Many packages are .aar files, but some are just plain jars

Reply:
Here's the list of .jar files that are in the latest templates NuGet cache that aren't signed - https://gist.github.com/pjcollins/737bfaa4df088098c4136479a686f5dc